README.md

```diff
@@ -184,10 +184,24 @@ RuntimeError: Calculated padded input size per channel: <smaller than 5 x 5>.
 Kernel size: (5 x 5). Kernel size can't be greater than actual input size
 ```
 
-#### Testing
+#### Inference
 The [`testing/main.py`](src/gz21_ocean_momentum/testing/main.py) script runs the
-model testing stage. You select a trained model and a region (which should be
-new/unseen) to test it on.
+model testing stage. This consists of running a trained model on a dataset. 
+The model's output are then stored as an artefact. This step
+should ideally be run with a GPU device available.
```

Comment:
Why should it be run on a GPU, and what happens when it is run on a CPU? Weirdly, I still get outputs saying that it is on a GPU even though I am running it on CPU, as the "device" variable is not checked/used in the print output.

Reply:
Hum it probably runs fine on a CPU, but might be quite slow. It is possible that the device checks are missing for the output.

Reply:
Added a comment in the readme about that